unit-tests/src/test/scala/java/util/PropertiesTest.scala

```diff
@@ -180,40 +180,40 @@ class PropertiesTest {
   @Test def non_string_values(): Unit = {
     val properties = new Properties
 
-    properties.put("age", Int.box(18))
+    properties.put("age", new Integer(18))
```

Comment:
The correct way to do this is in fact
```suggestion
    properties.put("age", Integer.valueOf(18))
```